src/test/java/pkmncore/PokemonTest.java

```diff
@@ -0,0 +1,26 @@
+package pkmncore;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class PokemonTest {
+
+    private Pokemon pokemon = new Pokemon("Pikachu", 1, new String[]{"lightning-rod", "static"});
+
+    @Test
+    public void hasAName() {
+        assertEquals("Pikachu", pokemon.getName());
+    }
+
+    @Test
+    public void hasAHeight() {
+        assertEquals(1, pokemon.getHeight());
+    }
+
+    @Test
+    public void hasAbilities() {
+        assertEquals("lightning-rod", pokemon.getAbilities()[0]);
```

Comment:
Could be worth checking the number of abilities returned is what is expected also.